src/Application/Actions/Charities/MarkUpdateRequiredFromSF.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Application\Actions\Charities;
+
+use Doctrine\ORM\EntityManagerInterface;
+use MatchBot\Application\Actions\Action;
+use MatchBot\Application\Assertion;
+use MatchBot\Domain\CharityRepository;
+use MatchBot\Domain\DomainException\DomainRecordNotFoundException;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Psr\Log\LoggerInterface;
+
+/**
+ * @psalm-suppress UnusedClass - to be added to routes.php, need to think about authorization requirements.
+ */
+class MarkUpdateRequiredFromSF extends Action
```

Comment:
this class feels so simple that it might not be worth automated testing. I can make a test if anyone wants though, or this might end up just being a used in a test (and therefore tested incidentally) for the process to actually pull the updates from SF after they get marked as required.